core/src/main/java/org/elasticsearch/index/engine/IndexSearcherWrapper.java

```diff
@@ -36,10 +36,12 @@
     DirectoryReader wrap(DirectoryReader reader);
 
     /**
-     * @param searcher The provided index searcher to be wrapped to add custom functionality
+     * @param searcher      The provided index searcher to be wrapped to add custom functionality
+     * @param engineConfig  The engine config which can be created to get the query cache and query cache policy from
+     *                      when creating a new index searcher
```

Comment:
Shouldn't the `@param engineConfig` come first in the javadocs, since it's first in the args?